exposee-ui/src/components/Mainpage/Main.jsx

```diff
@@ -0,0 +1,83 @@
+import "./main.css";
+import { useState, useEffect, useContext } from "react";
+import { UserContext } from "../../UserContext"; 
+import { Link } from "react-router-dom";
+
+function Main() {
+  const {user, updateUser} = useContext(UserContext);
+  const [videos, setVideos] = useState([]);
+  const [currentIndex, setCurrentIndex] = useState(0);
+
+  useEffect(() => {
+    const fetchVideos = async () => {
```

Comment:
ah so I think maybe I misunderstood what you meant by revert

you can keep Main.jsx in this PR, but get rid of the untested parts, so like we aren't using the video fetching, so you should remove them until you're ready to test them in a future PR